### PR TITLE
Add version fallback to ingestion

### PR DESCRIPTION
**Pull request by shans**
If the newest library version can't be ingested successfully, fallback on the next-most-recent version so that we have something in the search index.

### CODE REVIEW

File: src/manage.py [hunk 1]
```diff
@@ -82,15 +80,31 @@ def get(self, owner, repo, kind):
...
+        library.tags_etag = response.headers.get('ETag', None)
+        data.reverse()
+        is_newest = True
         for version in data:
           tag = version['ref'][10:]
           if not versiontag.is_valid(tag):
             continue
           sha = version['object']['sha']
+          params = {}
+          if (is_newest):
```
Comment (dstockwell) [inline src/manage.py]: excess parens

File: src/manage.py [hunk 2]
```diff
@@ -82,15 +80,31 @@ def get(self, owner, repo, kind):
           github.release()
           library.put()
           return
+        data = [d for d in data if versiontag.is_valid(d['ref'][10:])]
+        if len(data) is 0:
```
Comment (dstockwell) [inline src/manage.py]: merge these two error cases?, set data to [] above

File: src/manage.py [hunk 3]
```diff
@@ -137,18 +163,13 @@ def get(self, owner, repo, version):
     try:
       json.loads(response.content)
```
Comment (dstockwell) [inline src/manage.py]: FIXME? We need to abort (return 500) if the markdown request failed.

**Comment by dstockwell** [comment]
lgtm

File: src/manage.py (hunk 3 above)
Comment (shans) [inline src/manage.py]: No.

File: src/manage.py (hunk 1 above)
Comment (shans) [inline src/manage.py]: removed

File: src/manage.py (hunk 2 above)
Comment (shans) [inline src/manage.py]: done

**Pull request merged**